pkg/antctl/command_definition.go

```diff
@@ -135,15 +139,21 @@ func (e *resourceEndpoint) flags() []flagInfo {
 	if e.groupVersionResource == &v1beta2.NetworkPolicyVersionResource {
 		flags = append(flags, getSortByFlag())
 	}
+	if e.groupVersionResource == &v1beta2.AppliedToGroupVersionResource {
+		flags = append(flags, getSortByFlag())
+	}
+	if e.groupVersionResource == &v1beta2.AddressGroupVersionResource {
+		flags = append(flags, getSortByFlag())
+	}
 	return flags
 }
 
 func getSortByFlag() flagInfo {
 	return flagInfo{
 		name:            "sort-by",
 		defaultValue:    "",
-		supportedValues: []string{sortByEffectivePriority},
-		usage:           "Get NetworkPolicies in specific order. Current supported value is effectivePriority.",
+		supportedValues: []string{sortByEffectivePriority, sortByName, sortByCreationTimestamp, sortByUID, sortBySourceName},
+		usage:           "Get resources in specific order.",
```

Comment:
this is inaccurate.
In all 3 cases, arbitrary field paths are supported, or should be supported.
NetworkPolicy is the only resource which supports the special case of "effectivePriority".